public/app/plugins/datasource/loki/components/LokiExploreExtraField.tsx

```diff
@@ -1,33 +1,68 @@
 // Libraries
 import React, { memo } from 'react';
+import { css, cx } from 'emotion';
 
 // Types
-import { InlineFormLabel } from '@grafana/ui';
+import { InlineFormLabel, RadioButtonGroup } from '@grafana/ui';
 
 export interface LokiExploreExtraFieldProps {
-  label: string;
-  onChangeFunc: (e: React.SyntheticEvent<HTMLInputElement>) => void;
+  lineLimitValue: string;
+  queryType: string;
+  onLineLimitChange: (e: React.SyntheticEvent<HTMLInputElement>) => void;
   onKeyDownFunc: (e: React.KeyboardEvent<HTMLInputElement>) => void;
-  value: string;
-  type?: string;
-  min?: number;
+  onQueryTypeChange: (value: string) => void;
 }
 
 export function LokiExploreExtraField(props: LokiExploreExtraFieldProps) {
-  const { label, onChangeFunc, onKeyDownFunc, value, type, min } = props;
+  const { onLineLimitChange, onKeyDownFunc, lineLimitValue, queryType, onQueryTypeChange } = props;
+
+  const rangeOptions = [
+    { value: 'range', label: 'Range' },
+    { value: 'instant', label: 'Instant' },
+  ];
 
   return (
-    <div className="gf-form-inline">
-      <div className="gf-form">
-        <InlineFormLabel width={5}>{label}</InlineFormLabel>
+    <div aria-label="Loki extra field" className="gf-form-inline">
+      {/*Query type field*/}
+      <div
+        data-testid="queryTypeField"
+        className={cx(
+          'gf-form explore-input-margin',
+          css`
+            flex-wrap: nowrap;
+          `
+        )}
+        aria-label="Query type field"
+      >
+        <InlineFormLabel
+          tooltip="Choose the type of query you would like to run. Instant query allows doing queries against a single point in time. Range query is used to do a query over a range of time."
```

Comment:
@oddlittlebird Could you please look at this tooltip and share your thoughts if this is clear. Thank you. 🙏

Reply:
```suggestion
          tooltip="Choose the type of query you would like to run. An instant query queries against a single point in time. A range query queries over a range of time."
```

Reply:
How's that? If we can format the text in the tooltip, then we should make "instant" and "range" italic in both cases, or make them bold and turn them into bullets.

Reply:
Yes, I like that. Also in this label tooltips, it is not possible to style text.